polaris-core/src/main/java/org/apache/polaris/core/storage/aws/AwsStorageConfigurationInfo.java

```diff
@@ -94,7 +101,7 @@ public AwsStorageConfigurationInfo(
       @Nonnull String roleARN,
       @Nullable String externalId,
       @Nullable String region) {
-    this(storageType, allowedLocations, roleARN, externalId, region, null, null);
+    this(storageType, allowedLocations, roleARN, externalId, region, null, null, false);
```

Comment:
when we have `PATH_STYLE_ACCESS_DEFAULT` defined we can wire in that or just null here so that actual default get applied with Optional handling L87